### DISCUSSION TITLE
Give up button

### DISCUSSION THREAD

**Discussion by Billos** [bafolts/terraforming-mars#6581] [Ideas]
It doesn't happen a lot, but sometimes a player is way behind everybody else and doesn't care anymore, allowing him to just give up would waste less time for asynchronous multiplayer games

**Comment by kberg** [comment]
Thank you. Discussed in https://github.com/terraforming-mars/terraforming-mars/issues/93, https://github.com/terraforming-mars/terraforming-mars/pull/1377 and others
